apps/mobile-wallet/src/features/qrCodeScan/CameraScanButtonBase.tsx

```diff
@@ -28,7 +28,7 @@ const CameraScanButtonBase = ({
   generateInvalidDataText,
   ...props
 }: CameraScanButtonBaseProps) => {
-  const isFocused = useAppScreenIsFocused()
+  const isFocused = useScreenIsFocused()
```

Comment:
I think this component is also rendered inside a modal, so shouldn't we use the other hook?